paddlenlp/transformers/gpt/modeling.py

```diff
@@ -894,7 +911,7 @@ def forward(self, prediction_scores, masked_lm_labels, loss_mask):
 class GPTForGreedyGeneration(GPTPretrainedModel):
     """
     The generate model for GPT-2.
-    It use the greedy strategy and generate the next word with highest probability.
+    It use the greedy strategy and generate the next sequence with highest probability.
```

Comment:
the next sequence -> the output sequence

Reply:
done